files/es/web/javascript/index.md

```diff
@@ -0,0 +1,119 @@
+- [Cierres (clousures)](/es/docs/Web/JavaScript/Closures)
+  - : Un cierre es la combinación de una función y un entorno léxico dentro del cual se declaró esa función.
+
+### Avanzado
+
+- [Herencia y la cadena de prototipos](/es/docs/Web/JavaScript/Inheritance_and_the_prototype_chain)
+  - : Explicación de la herencia basada en prototipos ampliamente incomprendida y subestimada.
+- [Modo estricto](/es/docs/Web/JavaScript/Reference/Strict_mode)
+  - : El modo estricto define que no puedes usar ninguna variable antes de iniciarla. Es una variable restringida de ECMAScript 5, para un rendimiento más rápido y una depuración más fácil.
+- [Arreglos escritos en JavaScript](/es/docs/Web/JavaScript/Typed_arrays)
+  - : Los arreglos escritos en JavaScript proveen un mecanismo para acceder a datos binarios sin formato.
+- [Gestión de la Memoria](/es/docs/Web/JavaScript/Memory_Management)
+  - : Ciclo de vida de la memoria y recolección de basura en JavaScript.
+- [Modelo de Concurrencia y Bucle de Eventos](/es/docs/Web/JavaScript/EventLoop)
+  - : JavaScript tiene un modelo de concurrencia basado en un "bucle de eventos".
+
+## Referencia
+
+Consulta la documentación completa de la [referencia de JavaScript](/es/docs/Web/JavaScript/Reference).
+
+
+- [Objectos Estándar](/es/docs/Web/JavaScript/Reference/Global_Objects)
+  - : Conoce los objectos estándar integrados {{jsxref("Array")}}, {{jsxref("Boolean")}}, {{jsxref("Date")}}, {{jsxref("Error")}}, {{jsxref("Function")}}, {{jsxref("JSON")}}, {{jsxref("Math")}}, {{jsxref("Number")}}, {{jsxref("Object")}}, {{jsxref("RegExp")}}, {{jsxref("String")}}, {{jsxref("Map")}}, {{jsxref("Set")}}, {{jsxref("WeakMap")}}, {{jsxref("WeakSet")}}, y otros.
+- [Expresiones y operadores](/es/docs/Web/JavaScript/Reference/Operators)
+  - : Aprende más sobre el comportamiento de los operadores de JavaScript {{jsxref("Operators/instanceof", "instanceof")}}, {{jsxref("Operators/typeof", "typeof")}}, {{jsxref("Operators/new", "new")}}, {{jsxref("Operators/this", "this")}}, el [operador de precedencia](/es/docs/Web/JavaScript/Reference/Operators/Operator_Precedence ), y más.
+- [Sentencias y declaraciones](/es/docs/Web/JavaScript/Reference/Statements)
+  - : Aprende cómo funciona {{jsxref("Statements/do...while", "do-while")}}, {{jsxref("Statements/for...in", "for-in")}}, {{jsxref("Statements/for...of", "for-of")}}, {{jsxref("Statements/try...catch", "try-catch")}}, {{jsxref("Statements/let", "let")}}, {{jsxref("Statements/var", "var")}}, {{jsxref("Statements/const", "const")}}, {{jsxref("Statements/if...else", "if-else")}}, {{jsxref("Statements/switch", "switch")}}, y más declaraciones y palabras reservadas de JavaScript.
+- [Funciones](/es/docs/Web/JavaScript/Reference/Functions)
+  - : Aprende cómo trabajar con funciones de JavaScript para desarrollar tus aplicaciones.
+
+
```

Comment:
Another extra newline here **and** at line 96 (or 97, depending on which one you decide to remove)

Reply:
N.B. There are some "extra" single newlines between bullet points as well (e.g. line 57), for example:

- Text here

- More text here

I didn't comment on those occurrences since they technically shouldn't change the formatting/layout of the page in the end, so it really has to do with Markdown syntax/style and consistency

Reply:
@cw118 if you still helping us, you will learn Spanish :smile: